src/actions/profile-view.js

```diff
@@ -66,6 +66,15 @@ export function changeRightClickedThread(selectedThread: ThreadIndex): Action {
   };
 }
 
+export function setProfileCallTreeContextMenuVisibility(
+  isVisible: boolean
+): Action {
+  return {
+    type: 'SET_PROFILE_CALL_TREE_CONTEXT_MENU_VISIBILITY',
```

Comment:
This is a new action added to be able to disable the tooltips when the menu is open.